src/Drupal/StateTrait.php

```
@@ -0,0 +1,247 @@
+  /**
+   * Assert that a Drupal state value equals an expected value.
+   *
+   * @code
+   * Then the state "my_module.launched" should have the value "1"
+   * @endcode
+   */
+  #[Then('the state :name should have the value :value')]
+  public function stateAssertHasValue(string $name, string $value): void {
+    $actual = \Drupal::state()->get($name);
+    if ($actual === NULL) {
+      throw new \Exception(sprintf('The state "%s" does not exist, but it should have the value "%s".', $name, $value));
```

PHPMD (2.15.0)

[error] Missing class import via use statement (line '138', column '17'). (undefined)

(MissingImport)